code/switcher.py

```diff
@@ -180,7 +180,7 @@ def update_lists():
 
         words = get_words(name)
         for word in words:
-            if word and word not in running:
+            if word and word not in running and len(name) > 3:
```

Comment:
you need to check word, rather than name

Reply:
should probably maintain >= too

Reply:
Wow, I feel embarrassed to have moved that fix so incorrectly. Sorry about that!

Reply:
I would have tracked that down in my troubleshooting if I had separated out the "no commands are working" symptom from my previous efforts. Lessons learned.

Reply:
no worries, this happens to everyone. thanks for taking a look at this